src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,21 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post }) => (
```

Comment:
Ensure that the `post` prop is passed correctly to the `PostInfo` component. It should be an object containing `title`, `body`, `user`, and `comments` properties.